### PR TITLE
Update github icon color in the footer.

### PR DESCRIPTION
**Pull request by Astrodevil**
Here I changed the GitHub logo Color in the Follow us Section.  I made the logo color same as contact us color.
gw

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
     <link rel="shortcut icon" href="images/icon.png" type="images/icon.png" />
     <title>Algo Phantoms</title>
 
-    
+
```
Comment (harikesh409) [inline index.html:23]: remove blank line.

File: index.html [hunk 2]
```diff
@@ -155,7 +155,7 @@ <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Features<
                 </div>
                 <div class="col-lg-3 col-md-3 col-sm-6 col-12">
                     <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Follow Us</h5>
-                    <a href="https://github.com/Algo-Phantoms"><i class="fa fa-github p-2 text-white" aria-hidden="true" style="background: rgba(0, 0, 0, 0.15); border-radius: 50%;"></i>&nbsp;&nbsp;
+                    <a href="https://github.com/Algo-Phantoms"><i class="fa fa-github p-2 text-white" aria-hidden="true" style="background:#3caea3; border-radius: 50%;"></i>&nbsp;&nbsp;
```
Comment (harikesh409) [inline index.html:158]: Remove the `&nbsp;&nbsp;` which is causing unnecessary blue line on hover.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (Astrodevil) [reply index.html:158]: Ok I will do that

Comment (Astrodevil) [reply index.html]: I have done the changes please review

File: index.html (hunk 1 above)
Comment (Astrodevil) [reply index.html]: done

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (harikesh409) [inline .vscode/settings.json:1]: remove this file

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by harikesh409** [comment]
@Astrodevil resolve the merge conflicts.

**Comment by harikesh409** [comment]
@Astrodevil can you update the same color in `team.html` also so that the footer looks same across the whole site.

**Comment by Astrodevil** [comment]
Yea sure

**Pull request merged**